plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/HiveMetastoreTableOperations.java

```diff
@@ -95,7 +98,7 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
                 catch (RuntimeException ex) {
                     e.addSuppressed(ex);
                 }
-                throw e;
+                throw new TrinoException(ICEBERG_COMMIT_ERROR, format("Failed to commit to table %s.%s", database, tableName), e);
```

Comment:
Can simplify to
```java
throw new TrinoException(ICEBERG_COMMIT_ERROR, "Failed to commit to table: " + table.getSchemaTableName(), e);
```
Since `SchemaTableName` already formats nicely for error messages

Reply:
Can't log with `table` because it's defined in the try block, should be able to clean it up separately along with the other comment:

> This catch block is actually in the wrong place. The call most likely to fail is the replaceTable() call below, so this should really be on the outside. Though we could fix this separately.